app/src/main/java/com/nutomic/syncthingandroid/activities/SettingsActivity.java

```diff
@@ -303,6 +308,7 @@ public boolean onSyncthingPreferenceChange(Preference preference, Object o) {
                     mOptions.globalAnnounceServers = Iterables.toArray(splitter.split((String) o), String.class);
                     break;
                 case "address":               mGui.address = (String) o;  break;
+                case "restartOnWakeup":       mOptions.restartOnWakeup = (boolean) o;              break;
```

Comment:
Some sketchy formatting, either move the two options that span the same line above next to others (and then align the breaks), or align don't align the breaks at all, or align the breaks within the subsection.